concepts/80 Troubleshooting/50 A Drop-Down Editor Does Not Show Data/10 DataField and ValueExpr Should Store The Same Keys.md

```diff
@@ -0,0 +1,98 @@
+To correctly substitute displayed values with required values in a DataGrid/TreeList lookup column, follow these steps:
+
+1. Define the [lookup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/).[dataSource](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/#dataSource).
+
+2. Specify the [lookup](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/).[valueExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/#valueExpr). It should match the [columns](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/).[dataField](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#dataField). Refer to the following topic for detailed explanation: [lookup - API Section](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/).
+
+3. Use the [displayExpr](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/lookup/#displayExpr) property to specify displayed values.
```

Comment:
Try to use the strong verbs instead of "use". Not use, but set, define, specify, so a user can understand how exactly they can use this property without further explanation.